examples/k8s.yaml

```diff
@@ -54,10 +54,46 @@ provision:
     apt-get install -y cri-tools
     cat  <<EOF | sudo tee /etc/crictl.yaml
     runtime-endpoint: unix:///run/containerd/containerd.sock
-    image-endpoint: unix:///run/containerd/containerd.sock
     EOF
     # cni-plugins
     apt-get install -y kubernetes-cni
+    mkdir -p /etc/cni/net.d
+    cat << EOF | tee /etc/cni/net.d/10-containerd-net.conflist
```

Comment:
Looks like we are going to support multi-node example (https://github.com/lima-vm/lima/pull/1183), so probably we do not need to use non-flannel bridge?

Reply:
It would have to be versioned, at least. That is, only apply when number of nodes == 1. Otherwise flannel.

Reply:
Is it harmless to use flannel on single-node mode too?

Reply:
Yeah, just overkill. Mostly it was triggered by the kubeadm team not wanting to support flannel anymore.